src/mlpack/methods/adaboost/adaboost_main.cpp

```diff
@@ -230,7 +243,11 @@ static void mlpackMain()
     Row<size_t> results;
     data::RevertLabels(predictedLabels, m->Mappings(), results);
 
-    CLI::GetParam<arma::Row<size_t>>("output") = std::move(results);
+    // Save the predicted labels.
+    if (CLI::HasParam("output"))
+      CLI::GetParam<arma::Row<size_t>>("output") = std::move(results);
+    if (CLI::HasParam("predictions"))
+      CLI::GetParam<arma::Row<size_t>>("predictions") = std::move(results);
```

Comment:
We can't `std::move(results)` twice---I'd suggest only doing it the second time.

Reply:
Done